### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by AmayX** [carbon-language/carbon-lang#2674] [General]
Hey everyone
I would like to contribute to creating documentation for Carbon Lang for GSoC 2023. Please guide me about the same, @jonmeow

**Comment by EzzalddeenAli** [comment]
I think it is best to create the documentation when the first beta version of the language is completed. During this period we need to understand the language and contribute to its development

**Reply by EzzalddeenAli** [reply]
It is also preferable to start creating documentation for language development

**Comment by jonmeow** [comment]
@AmayX  Please see [Carbon's GSoC contributor guidance](https://docs.google.com/document/d/1Ly3tD58IE57h_xsoZ8wsL8y4Mp89fg8J8qLN8sCPZ5k/preview?resourcekey=0-CLlGUVSc7DMhLPTABpmjRQ).

**Comment by CatX711** [comment]
@AmayX you also might want to add to the Wikipedia page, it would make info about Carbon more accessible for beginners and just in general people interested in the language.
